### ISSUE TITLE
Clean install misses perl-LWP-Protocol-https.noarch

### ISSUE THREAD

**Issue by igarny** [perfsonar/graphs#135]
Hello Michael,

For me a clean install of **perfsonar-centralmanagement** on a CentOS7.5 misses 
perl-LWP-Protocol-https.noarch

the message on the dashboard is: 
```
Protocol scheme 'https' is not supported (LWP::Protocol::https not installed)
```

**Comment by igarny** [comment]
I have today verified the issue with another clean install

**Comment by igarny** [comment]
![image](https://user-images.githubusercontent.com/11942774/48761106-1ffaef80-eca8-11e8-9bf2-d3c4c4775ee9.png)

**Comment by arlake228** [comment]
We had a similar issue on the toolkit we fixed in 4.1.3. I believe one of the third-party libraries previously had a dependency on this which got removed and since we don't have a direct dependency this is giving us problems now. Will fix the dependency in 4.1.4 for the graphs.

**Comment by arlake228** [comment]
See my comment in #136. This was indeed a separate issue but it is resolved by the same fix.
